app/models/site_config.rb

```diff
@@ -26,26 +26,25 @@ class SiteConfig < RailsSettings::Base
   field :campaign_articles_require_approval, type: :boolean, default: 0
 
   # Community Content
-  field :community_description, type: :string, default: "A constructive and inclusive social network. Open source and radically transparent."
-  field :community_member_description, type: :string, default: "amazing humans who code."
```

Comment:
Removed community_member_description as I think its repetitive. We used  community_member_description only on the sign in invitation banner and it was configured to read `DEV is a community of x amazing humans who code`, community_member_description had a value of `amazing humans who code`. 

Instead I've amended the sentence to use the community_member_label and changed it to `DEV(local) is a community of 11 amazing developers`. 

I feel like reusing config for these kind of things makes more sense than to add some more.